packages/alfa-rules/src/sia-r54/rule.ts

```diff
@@ -0,0 +1,54 @@
+import { Rule, Diagnostic } from "@siteimprove/alfa-act";
+import { Element } from "@siteimprove/alfa-dom";
+import { Predicate } from "@siteimprove/alfa-predicate";
+import { Err, Ok } from "@siteimprove/alfa-result";
+import { Page } from "@siteimprove/alfa-web";
+import { isIgnored } from "../common/predicate/is-ignored";
```

Comment:
```suggestion

import { isIgnored } from "../common/predicate";
```
`alfa-rules/src/common/predicate.ts` re-exports all predicates, so it's often easier to take them from here.